template/admin/initServer.sqf

```diff
@@ -6,17 +6,13 @@ publicVariable QMGVAR(admins);
 MGVAR(debug_admins) = [DEBUG_ADMINS];
 publicVariable QMGVAR(debug_admins);
 
-// Calculate time until start (0 if debug mode enabled)
-systemTimeUTC params ["", "", "", "_hour", "_minute"];
-private _startTime = 14; // 1400z always
-GVAR(timeUntilStart) = ((_startTime * 60) - (_hour * 60 + _minute)) * 60; // start time - current time = time until start time
-GVAR(timeUntilStart) = GVAR(timeUntilStart) max 0;
+// Set time until start to 0 if debug mode is enabled.
+[] call MFUNC(timeUntilStart);
 
 #ifdef DEBUG_MODE_FULL
 GVAR(timeUntilStart) = 0;
 #endif
```

Comment:
This is going to be different on every machine when debug mode is enabled, because you broadcast it before setting it to 0.

Instead of this, I would just do:
```sqf
// Set time until start (0 if debug mode enabled)
#ifdef DEBUG_MODE_FULL
GVAR(timeUntilStart) = 0;
#else
GVAR(timeUntilStart) = call MFUNC(timeUntilStart);
#endif

publicVariable QGVAR(timeUntilStart);
INFO_1("Time Until Start: %1",GVAR(timeUntilStart));
```
So `MFUNC(timeUntilStart)` only returns the calculated time. This also avoids using `tac_scripts_` prefix in Mods.